### DISCUSSION TITLE
Old/New Cross Architecture Supporting Table, is it correct?

### DISCUSSION THREAD

**Discussion by mym0404** [reactwg/react-native-new-architecture#168] [Q&A]
Hello, I read all posting in this docs and want to ensure my understanding is correct.

I drew a table about that. is it correct?

![image](https://github.com/reactwg/react-native-new-architecture/assets/33388801/970b1f07-b393-479d-95d2-0964fbdbb926)

Any wrong point or feedback is welcome!

**Comment by cortinico** [comment]
What does "Bridge with/without support bridgeless interop" means in your table?

**Reply by mym0404** [reply]
It means the inner APIs of library are migrated for Bridgeless discussed in [Bridgeless - 3. Migrate legacy APIs](https://github.com/reactwg/react-native-new-architecture/discussions/154).

Honestly, I am not sure whether the [3. Migrate legacy APIs]. Migrate legacy APIs is a change that should be applied to the app or a change that should be applied to the library.

**Reply by mym0404** [reply]
Even recently, I haven't seen catalystInstance actually used in any app or library.
